src/java/htsjdk/samtools/util/SamLocusIterator.java

```diff
@@ -353,9 +395,17 @@ private void accumulateSamRecord(final SAMRecord rec) {
         final int alignmentEnd    = rec.getAlignmentEnd();
         final int alignmentLength = alignmentEnd - alignmentStart;
 
+        // get the offset for an insertion if we are tracking them
```

Comment:
I extracted the computation of the offset for the queue in  the method `getAccumulatorOffset` in commit 7c9be2b1d221233cadb5180b50984e6760888b87